barretenberg/cpp/src/barretenberg/vm/tests/avm_arithmetic.test.cpp

```diff
@@ -334,7 +334,7 @@ TEST_F(AvmArithmeticTestsFF, addition)
     EXPECT_EQ(alu_row.avm_alu_cf, FF(0));
     EXPECT_EQ(alu_row.avm_alu_u8_r0, FF(0));
 
-    validate_trace_proof(std::move(trace));
+    validate_trace_check_circuit(std::move(trace));
```

Comment:
any changes in the test files are renaming to validate_trace_proof to validate_trace_check_circuit